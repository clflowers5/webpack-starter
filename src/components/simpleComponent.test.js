import React from 'react';
import { shallow } from 'enzyme';
import SimpleComponent from './simpleComponent';

describe('simpleComponent', () => {
  it('snapshot', () => {
    const wrapper = shallow(<SimpleComponent message="testing123" />);
    expect(wrapper).toMatchSnapshot();
  });
});
