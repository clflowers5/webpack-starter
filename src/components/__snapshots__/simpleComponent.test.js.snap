// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simpleComponent snapshot 1`] = `
<div>
  <p>
    testing123
  </p>
</div>
`;
